support-frontend/assets/helpers/redux/checkout/address/state.ts

```diff
@@ -46,9 +46,10 @@ export type AddressFieldsState = AddressFields & {
 };
 
 export function getInitialAddressFieldsState(): AddressFieldsState {
+	const country = detect();
 	return {
-		country: detect(),
-		state: '',
+		country,
```

Comment:
Tiny note but could revert creating a country var here for consistency with the line below

Reply:
I’m sorry, I don’t follow – are you suggesting removing the `country` var and calling `detect()` twice?

Reply:
Ahh I'd missed if there was a second use of `country` var! Best not to call `detect()` twice 👍